src/classes/HotkeysUtil.js

```diff
@@ -10,18 +10,18 @@ export default class HotkeysUtil {
       zoom: 'Zoom',
       angle: 'Angle',
       dragProbe: 'Pixel Probe',
-      ellipticalRoi: 'Elliptical ROI',
-      rectangleRoi: 'Rectangle ROI',
+      ellipticalRoi: 'EllipticalRoi',
```

Comment:
Is the second value here the display name? If yes, can we match the display name in the meteor version?

![image](https://user-images.githubusercontent.com/5797588/57941417-e395fe00-789c-11e9-9df1-d06b7d566fc1.png)

Reply:
Not it is not. The display name comes from `reducers/preferences.js` https://github.com/OHIF/ohif-core/blob/master/src/redux/reducers/preferences.js#L5